### PR TITLE
TF-11115, TF-11117, TF-11116] Adding object store connection string to output

### PR DESCRIPTION
**Pull request by kkavish**
Adding object store connection string to output

[Jira TF-11115](https://hashicorp.atlassian.net/browse/TF-11115)
[Jira TF-11116](https://hashicorp.atlassian.net/browse/TF-11116)
[Jira TF-11117](https://hashicorp.atlassian.net/browse/TF-11117)

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -52,7 +52,7 @@ output "storage_account_container_name" {
 }
 
 output "storage_account_primary_blob_connection_string" {
-  value       = module.object_storage.storage_account_primary_blob_connection_string
+  value       = local.object_storage.storage_account_primary_blob_connection_string
```
Comment (nikolasrieble) [inline outputs.tf:55]: 💭 Interesting, could you please explain why this change is needed? Does it currently not work?

Comment (kkavish) [reply outputs.tf:55]: following the pattern + to avoid mentioning index in a list.

**Review by nikolasrieble — APPROVED** [review]

File: outputs.tf (hunk 1 above)
Comment (nikolasrieble) [reply outputs.tf:55]: We explicitly mention that this repo is a beta, and it is not versioned, but I just want to call out that this would break existing usage of the output, right?

We are fine with it, but really we should just version the repo.
